samples/AsyncQuickInfoDemo/AsyncQuickInfoDemo.VSIX/Properties/AssemblyInfo.cs

```diff
@@ -1,14 +1,15 @@
 ﻿using System.Reflection;
+using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
 // General Information about an assembly is controlled through the following 
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
-[assembly: AssemblyTitle("AsyncQuickInfoDemo.VSWindows")]
```

Comment:
nit: VSIX is a lot less transparent than 'vswindows' to those who haven't extended VS before.

Reply:
Won't fix - AsyncCompletion sample uses .Mac and .VSIX suffixes.